File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int cnt = 0;
        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) != ' '){ // s값중에 공백이 없을경우
                if(cnt%2==1){ // cnt가 홀수일 경우
                    answer += Character.toLowerCase(s.charAt(i));
                }else{
                    answer += Character.toUpperCase(s.charAt(i));
                }
                cnt++;
            }else{ // s값중에 공백이 있을경우
                answer += " ";
                cnt = 0;
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s){
  let result = "";

  for(let word of s.split(" ")) {
    for(let i in word) {
      result += word[i][parseInt(i) % 2 == 0 ? "toUpperCase" : "toLowerCase"]();
    }
    result += " ";
  };

  return result.slice(0, -1);
}
